frequencyChar.js

```diff
@@ -0,0 +1,20 @@
+function itemFrequency(array) {
+    let characterFrequency = {};
+    let totalCount = 0
+  
+    for (let item of array) {
```

Comment:
You are already doing this work in `itemCounts`. Use `require` to load `itemCounts` and use that function instead of duplicating it line-by-line here.

For example, if you ever change something about how `itemCounts` works, you'll have to remember to change it here, too.